### PR TITLE
Secure fluent metrics endpoint

### PR DESCRIPTION
**Pull request by jcantrill**
fix https://bugzilla.redhat.com/show_bug.cgi?id=1615237 by

* removing cluster leaking metrics (e.g. in_tail paths)
* secure the endpoint

### CODE REVIEW

File: fluentd/Dockerfile [hunk 1]
```diff
@@ -52,6 +52,7 @@ ADD parser_viaq_docker_audit.rb viaq_docker_audit.rb /etc/fluent/plugin/
 ADD run.sh generate_syslog_config.rb ${HOME}/
 ADD lib/generate_throttle_configs/lib/*.rb ${HOME}/
 ADD lib/filter_parse_json_field/lib/*.rb /etc/fluent/plugin/
+ADD patch/in_prometheus.rb ${GEM_HOME}/gems/fluent-plugin-prometheus-0.4.0/lib/fluent/plugin/
```
Comment (richm) [inline fluentd/Dockerfile]: This may be in a different location downstream

Comment (jcantrill) [reply fluentd/Dockerfile]: Updated GEM_HOME TO /usr/share/gems

**Comment by jcantrill** [comment]
@richm The upstream PR merged and I asked for a new release.  We'll see if that happens in a timely manor to replace the patch portions of this PR.

**Pull request merged**